client/src/components/VideoPlayer.tsx

```diff
@@ -18,11 +20,18 @@ export const VideoPlayer: React.FunctionComponent<VideoPlayerProps> = ({
         }
     }, [stream])
 
+    useEffect(() => {
+        const [ videoTrack ] = stream.getVideoTracks()
+        const [ audioTrack ] = stream.getAudioTracks()
+
+        videoTrack.enabled = activeCamera
+        audioTrack.enabled = activeMicrophone
+    }, [activeCamera, activeMicrophone])
```

Comment:
this effect should have more logic regarding to starting video or audio related to the `activeCamera` and `ActiveMicriphone`